src/module/system/damage/helpers.ts

```diff
@@ -34,6 +44,99 @@ const DamageCategorization = {
     },
 } as const;
 
+const faces = [4, 6, 8, 10, 12];
```

Comment:
All caps? `faces` is shadowed immediately below, which is confusing.

Reply:
Good catch, I hadn't noticed that.